dotcom-rendering/src/types/frontend.ts

```diff
@@ -105,4 +105,14 @@ export interface CAPIArticleType {
 	selectedTopics?: Topic[];
 
 	promotedNewsletter?: Newsletter;
+	tableOfContents?: TOCType[];
```

Comment:
I saw in another PR (I think an AR one) that this data was called 'outline'. Maybe we can align on this? No strong feelings here, we can keep one or the other or even stick with things as they are, just a thought really